src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_AssignmentOperator.cs

```diff
@@ -309,20 +311,31 @@ private BoundExpression VisitAssignmentOperator(BoundAssignmentOperator node, bo
                     rewrittenRight);
             }
 
-            // We have already lowered each argument, but we may need some additional rewriting for the arguments,
-            // such as generating a params array, re-ordering arguments based on argsToParamsOpt map, inserting arguments for optional parameters, etc.
-            ImmutableArray<LocalSymbol> argTemps;
-            rewrittenArguments = MakeArguments(
+            arguments = VisitArguments(
```

Comment:
>VisitArguments [](http://example.com/codeflow?start=24&length=14)

Was there a bug here that we never lowered aguments?

Reply:
No. Arguments were visited earlier, in LocalRewriter_IndexerAccess.cs line 85, but then we held off on calling MakeArguments until this point. However, we now need the output from VisitArguments for the MakeArguments call, so I pushed the visit down to the usage points.